logic/atrium-logic-common/src/main/kotlin/ch/tutteli/atrium/logic/impl/DefaultIterableLikeAssertions.kt

```diff
@@ -134,22 +122,6 @@ class DefaultIterableLikeAssertions : IterableLikeAssertions {
             }
         }
 
-    private fun <E> createIndexAssertions(
-        list: List<E>,
-        predicate: (WithIndex<E>) -> Boolean
-    ) = list
-        .asSequence()
-        .mapWithIndex()
-        .filter { predicate(it) }
-        .map { (index, element) ->
-            assertionBuilder.createDescriptive(
-                TranslatableWithArgs(DescriptionIterableAssertion.INDEX, index),
-                element,
-                falseProvider
-            )
-        }
-        .toList()
```

Comment:
I saw no reason to rewrite this logic so I moved this method into `containsHelpers.kt`